src/Compiler/CompilerDefines.h

```diff
@@ -51,28 +51,30 @@ Design::Language FromFileType(const QString &type);
 
 // ID of the tasks shouln't be changed since they save to file
 static constexpr uint IP_GENERATE{0};
-static constexpr uint SYNTHESIS{1};
-static constexpr uint SYNTHESIS_CLEAN{2};
-static constexpr uint SYNTHESIS_SETTINGS{3};
-static constexpr uint SYNTHESIS_WRITE_NETLIST{4};
-static constexpr uint SYNTHESIS_TIMING_REPORT{5};
-static constexpr uint PACKING{6};
-static constexpr uint PACKING_CLEAN{7};
-static constexpr uint GLOBAL_PLACEMENT{8};
-static constexpr uint GLOBAL_PLACEMENT_CLEAN{9};
-static constexpr uint PLACEMENT{10};
-static constexpr uint PLACEMENT_CLEAN{11};
-static constexpr uint PLACEMENT_SETTINGS{12};
-static constexpr uint PLACEMENT_WRITE_NETLIST{13};
-static constexpr uint PLACEMENT_TIMING_REPORT{14};
-static constexpr uint ROUTING{15};
-static constexpr uint ROUTING_CLEAN{16};
-static constexpr uint ROUTING_SETTINGS{17};
-static constexpr uint ROUTING_WRITE_NETLIST{18};
-static constexpr uint TIMING_SIGN_OFF{19};
-static constexpr uint POWER{20};
-static constexpr uint BITSTREAM{21};
-static constexpr uint PLACE_AND_ROUTE_VIEW{22};
+static constexpr uint ANALYSIS{1};
```

Comment:
this will brake previous saved files because we save task state into the project file. So, task ID shouldn't be changed. The order in the table is defined in the table model.